doc/source/_toc.yml

```diff
@@ -158,6 +158,7 @@ parts:
               - file: rllib/rllib-sample-collection
               - file: rllib/rllib-offline
               - file: rllib/rllib-dev
+              - file: rllib/rllib-replay-buffers
```

Comment:
Should we move this up a bit? E.g. after sample collection? I feel like in order to understand offline, you should have some knowledge on what a replay buffer is first.

Also: @maxpumperla ^